win32.mak

```diff
@@ -534,4 +534,9 @@ install: phobos.zip
 
 auto-tester-build: targets
 
-auto-tester-test: unittest
+JOBS=4
```

Comment:
Is there no handy way to detect this on Windows in a bash-like environment?
`nproc` is typically part of the coreutils?